portfolio/src/main/webapp/index.html

```diff
@@ -15,15 +15,19 @@
         </ul>
     </div>
     <div id="content">
-      <h1>Dean's Portfolio</h1>
-      <div id="aboutMe">
-        <h2>About me</h2>
+    	<h1>About me</h1>
         <p> My name is Dean Alvarez. I am a student at Cornell University studying
         Computer Science. My hobbies include listening to music, playing video games,
         talking with friends, and playing the piano.</p>
-      </div>
-      <p> Here is one of my favorite pieces of artwork: </p>
-      <img src="/images/GreatWave.jpeg"></img>
+    	<img src="/images/GreatWave.jpg" id="greatWave"></img>
+        <br>
+        <p>Connect:</p>
+    	<div class="row">
+            <div class="col"><a href="https://github.com/DeanAlvarez">
+              <img src="/images/GitHub-Mark-120px-plus.png" class= "connect"></img></a></div>
+      		<div class="col"><a href="https://www.linkedin.com/in/deanedwardalvarez">
+              <img src="/images/LI-In-Bug.png" class="connect" id="in"></img></a></div>
+        </div>
```

Comment:
It looks like you're using divs to make rows and columns. I would suggest checking out CSS grid, it makes accomplishing the layout a lot easier.